include/image.mk

```diff
@@ -572,6 +593,8 @@ define Device/Build/artifact
 endef
 
 define Device/Build
+  $(if $(CONFIG_JSON_ADD_IMAGE_INFO), $(shell rm -f $(BIN_DIR)/$(IMG_PREFIX)-$(1).json))
```

Comment:
Why only remove if config is set? If there are old json files there, they are old, so they should be removed independent of whether you create new ones ...

Reply:
Done